lib/batch.js

```diff
@@ -12,7 +5,7 @@ var debug = require('debug')('wpcom:batch');
  * @api public
  */
 
-function Batch(wpcom) {
+function Batch(wpcom) { //eslint-disable-line space-before-function-paren
```

Comment:
Good suggestion @aduth. When I introduced this option I had actually tested with that setting, but found something like twenty errors. Of course, this was before I dug in to realize what was actually happening and just assumed we didn't follow this convention.

Updated in the PR